.github/workflows/rust-checks.yml

```diff
@@ -51,7 +51,17 @@ jobs:
       
       - name: Cache dependencies
         uses: Swatinem/rust-cache@v2
-      
+
+      - name: Install nvcc
```

Comment:
Would this make sense in the GH CI? Since we can't do GPU proving tests, no?

I was thinking some time ago if we ever have some idle and cheap GPU prover machine, we could install the GH runner there and have some special CI workflow for GPU proving quick checks.

But until that happens not sure if installing nvcc makes sense? (maybe related on why the CI is failing)

Reply:
Because the CI does `cargo clippy --all-features` so some of the dependencies of `risc0` has the `build.rs` script to build some cuda code, so I install `nvcc` to make the clippy happy.

I was thinking to remove the `--all-features` but I'm not sure if it's a good idea or not, so I temporarily disabled the `cuda` feature of `ere-risc0` so bypass that, since in https://github.com/eth-act/ere/pull/50 we will upgrade it to use docker the do the multi-gpu proving.

I can also remove the `--all-features` if there is no objection, sinice I didn't much features flags among `ere-zkvm` crates.

Reply:
Then I think it'd be fine to remove `--all-features` because only `risc0` has feature flag.